### DISCUSSION TITLE
RC.0: theme-hope packages not compatible

### DISCUSSION THREAD

**Discussion by digital-codes** [vuepress-theme-hope/vuepress-theme-hope#3638] [Q&A] [answered]
First of all I'm happy to see you moving towards the release version. Thanks.
However, during the past beta versions the theme-hope packages like vuepress-plugin-sitemap2 have been provided in a compatible version. This seems not to be the case with rc.0  Would be nice to have those available to test rc.0

**Comment by Mister-Hope** [comment]
Reproduction about what you are talking is required

**Reply by digital-codes** [reply]
... could not find the proper version for the theme-hope package plugin-sitemap2. solved via vp-update. No issue any more.

**Reply by Mister-Hope** [reply, marked as answer]
It takes time for the community to compact upstream changes
